app/src/main/java/com/lagradost/cloudstream3/MainActivity.kt

```diff
@@ -607,6 +615,10 @@ class MainActivity : AppCompatActivity(), ColorPickerDialogListener {
             initAll()
             // No duplicates (which can happen by registerMainAPI)
             apis = allProviders.distinctBy { it }
+
+            if (githubApi.getLatestLoginData() != null && settingsManager.getBoolean(getString(R.string.automatic_cloud_backups), false)){
```

Comment:
You cannot have false as a default value here, but true in the xml
That will make it look like you have it on while it evaluates to false